components/TitleBar.js

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+
+const Headline = styled.h1`
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  background-color: white;
+  margin: 0;
+  padding: 20px;
+  text-align: center;
+  z-index: 1;
+`;
+
+export default function TitleBar() {
+  return <Headline>Plant Pal</Headline>;
+}
```

Comment:
instead of exporting this "wrapper" component, you can export the styled component `Headline` instead. In the Layout Component you then replace `<Titlebar />` With `<Headline>Plant Pal</Headline>`